### PR TITLE
Cherry picked 1.9.0 beta.1 into dev

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
These are the release changes for 1.9.0-beta.1 that have been merged in to development branch for issue #981

### CODE REVIEW

**Comment by ddimatos** [comment]
Ansible sanity:
```
~/git/gh/ibm_zos_core $: ./ac --ac-sanity
-----------------------------------------------------------------------
Running ansible-test with docker container and all python versions.
-----------------------------------------------------------------------
Starting new "ansible-test-controller-mDyo3kZH" container.
Adding "ansible-test-controller-mDyo3kZH" to container database.
Running sanity test "action-plugin-docs"
Running sanity test "ansible-doc"
Running sanity test "changelog"
Running sanity test "compile" on Python 2.7
Running sanity test "compile" on Python 3.5
Running sanity test "compile" on Python 3.6
Running sanity test "compile" on Python 3.7
Running sanity test "compile" on Python 3.8
Running sanity test "compile" on Python 3.9
Running sanity test "compile" on Python 3.10
Running sanity test "compile" on Python 3.11
Running sanity test "empty-init"
Running sanity test "future-import-boilerplate"
Running sanity test "ignores"
Running sanity test "import" on Python 2.7
Running sanity test "import" on Python 3.5
Running sanity test "import" on Python 3.6
Running sanity test "import" on Python 3.7
Running sanity test "import" on Python 3.8
Running sanity test "import" on Python 3.9
Running sanity test "import" on Python 3.10
Running sanity test "import" on Python 3.11
Running sanity test "line-endings"
Running sanity test "metaclass-boilerplate"
Running sanity test "no-assert"
Running sanity test "no-basestring"
Running sanity test "no-dict-iteritems"
Running sanity test "no-dict-iterkeys"
Running sanity test "no-dict-itervalues"
Running sanity test "no-get-exception"
Running sanity test "no-illegal-filenames"
Running sanity test "no-main-display"
Running sanity test "no-smart-quotes"
Running sanity test "no-unicode-literals"
Running sanity test "pep8"
Running sanity test "pslint"
Running sanity test "pylint"
Running sanity test "replace-urlopen"
Running sanity test "runtime-metadata"
Running sanity test "shebang"
Running sanity test "shellcheck"
Running sanity test "symlinks"
Running sanity test "use-argspec-type-path"
Running sanity test "use-compat-six"
Running sanity test "validate-modules"
Running sanity test "yamllint"
~/git/gh/ibm_zos_core $:
``` 

ansible-lint producs the 27 issues until the ignore entries are complete:
```
tests/sanity/ignore-2.14.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:7: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 7, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:14: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 14, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:25: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 25, which is not a permitted ignore.
tests/sanity/ignore-2.14.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 26, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:7: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 7, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:14: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 14, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:25: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 25, which is not a permitted ignore.
tests/sanity/ignore-2.15.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 26, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:5: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 5, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:6: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 6, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:7: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-choices-do-not-match-spec at line 7, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:8: sanity[cannot-ignore]: Ignore file contains validate-modules:doc-type-does-not-match-spec at line 8, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:10: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 10, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:13: sanity[cannot-ignore]: Ignore file contains validate-modules:parameter-type-not-in-doc at line 13, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:14: sanity[cannot-ignore]: Ignore file contains validate-modules:undocumented-parameter at line 14, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:25: sanity[cannot-ignore]: Ignore file contains validate-modules:invalid-extension at line 25, which is not a permitted ignore.
tests/sanity/ignore-2.16.txt:26: sanity[cannot-ignore]: Ignore file contains validate-modules:python-syntax-error at line 26, which is not a permitted ignore.
```

**Review by richp405 — APPROVED** [review]
wow!  doc mania, and a few code tweaks.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,42 @@ ibm.ibm_zos_core Release Notes
...
+- zos_job_submit - The module handling ZOAU import errors obscured the original traceback when an import error ocurred. Fix now passes correctly the context to the user. (https://github.com/ansible-collections/ibm_zos_core/pull/1091).
+- zos_mvs_raw - when using the dd_input content option for instream-data, if the content was not properly indented according to the program which is generally a blank in columns 1 & 2, those columns would be truncated. Now, when setting instream-data, the module will ensure that all lines contain a blank in columns 1 and 2 and add blanks when not present while retaining a maximum length of 80 columns for any line. This is true for all content types; string, list of strings and when using a YAML block indicator. (https://github.com/ansible-collections/ibm_zos_core/pull/1057). - zos_mvs_raw - no examples were included with the module that demonstrated using a YAML block indicator, this now includes examples using a YAML block indicator.
```
Comment (fernandofloresg) [inline CHANGELOG.rst:29]: I think this has two changes in one line.

Comment (ddimatos) [reply CHANGELOG.rst:29]: Thanks , good catch, odd because that was generated automatically.

Comment (ddimatos) [reply CHANGELOG.rst:29]: Fixed :)

**Review by fernandofloresg — APPROVED** [review]

**Pull request merged**